src/mon/MDSMonitor.cc

```diff
@@ -356,12 +356,12 @@ bool MDSMonitor::preprocess_beacon(MMDSBeacon *m)
  ignore:
   // note time and reply
   _note_beacon(m);
+ out:
```

Comment:
I don't think moving the out: label up here is correct. Everybody who goes to out actually wants to ignore the MDS in question — because it's destined for a different filesystem, or comes from an entity without the right caps, or something. Given that we need to send a reply to peon monitors, maybe we need a new ack_route message or something that will make them drop it. :/